CHANGELOG.md

```diff
@@ -1,11 +1,31 @@
+<!-- markdownlint-configure-file { "blanks-around-headers": { "lines_below": 0 } } -->
+<!-- markdownlint-configure-file { "blanks-around-lists": false } -->
```

Comment:
I think we should stick to the already defined Markdown linting rules defined in this project, but add exceptions for duplicated headers in `CHANGELOG.md` (cf. https://github.com/itk-dev/naevnssekretariatet/actions/runs/3427044717/jobs/5709564181).

Reply:
hvordan gør man det?